flink-dist/src/main/flink-bin/bin/taskmanager.sh

```diff
@@ -48,8 +48,9 @@ if [[ $STARTSTOP == "start" ]] || [[ $STARTSTOP == "start-foreground" ]]; then
 
     # Startup parameters
 
-    jvm_params_output=`runBashJavaUtilsCmd GET_TM_RESOURCE_JVM_PARAMS ${FLINK_CONF_DIR}`
-    jvm_params=`extractExecutionParams "$jvm_params_output"`
+    params_output=$(runBashJavaUtilsCmd GET_TM_RESOURCE_PARAMS ${FLINK_CONF_DIR} | tail -n 2)
+
+    jvm_params=$(extractExecutionParams "$(echo "$params_output" | tail -n 2 | head -n 1)")
```

Comment:
`head -n 1` should be sufficient since `params_output` already only contains 2 lines.